command/volume_status_csi.go

```diff
@@ -151,3 +153,43 @@ func (c *VolumeStatusCommand) formatTopologies(vol *api.CSIVolume) string {
 
 	return strings.Join(out, "\n")
 }
+
+func csiVolMountOption(volume, request *api.CSIMountOptions) string {
+	var vol, req, opts *structs.CSIMountOptions
+
+	if volume != nil {
+		vol = &structs.CSIMountOptions{
+			FSType:     volume.FSType,
+			MountFlags: volume.MountFlags,
+		}
+	}
+
+	if request != nil {
+		req = &structs.CSIMountOptions{
+			FSType:     request.FSType,
+			MountFlags: request.MountFlags,
+		}
+	}
+
+	if volume == nil {
+		opts = req
+	} else {
+		opts = vol.Copy()
```

Comment:
We don't use `vol` anywhere except in this block. It might make the nil-checking more obvious here if we moved the constructor for `vol` here?

Reply:
fixed!